src/controllers/movie.js

```diff
@@ -0,0 +1,108 @@
+const {
+    getMoviesDB,
+    createMovieDB,
+    getMovieByIdDB,
+    getMovieByTitleDB,
+    updateMovieDB,
+} = require("../domains/movie.js");
+const { get } = require("../routers/movie.js");
+
+const getMovies = async (req, res) => {
+    const filter = {
+        title: req.query.title,
+        runtimeMins: req.query.runtimeMins,
+        runtimeLt: req.query.runtimeLt,
+        runtimeGt: req.query.runtimeGt,
+    };
+    console.log(filter);
+    const movies = await getMoviesDB(filter);
+    res.status(200).json({ movies: movies });
+};
+
+const createMovie = async (req, res) => {
+    try {
+        const { title, runtimeMins } = req.body;
+        const filter = { title: title };
```

Comment:
When the property has the same name as the variable you're taking the value from, you don't need to write both:

`const filter = { title };`

Reply:
I seem to have a bad habit of this. I think I am doing it for clarity but never return to refactor.